lib/Dancer2/CLI/Command/gen.pm

```diff
@@ -265,20 +266,20 @@ sub _version_check {
     return if $version =~  m/_/;
 
     my $latest_version = 0;
-    my $resp = _send_http_request('http://search.cpan.org/api/module/Dancer2');
+    my $resp = _send_http_request( 'https://fastapi.metacpan.org/module/Dancer2' );
```

Comment:
I'd `s/module/release/` here  (though it really doesn't matter.. think the data returned is the same).

Reply:
Thanks for the tip! Changed.